.github/workflows/ci.yml

```diff
@@ -80,3 +80,51 @@ jobs:
           tag="${GITHUB_REF#refs/tags/}"
           args=$(printf -- "-a %q " {xtract-core,testing,macros}/target/*/*.jar)
           hub release create $args -m "Release $tag" $tag
+  publish-docs:
+    if: startsWith(github.ref, 'refs/tags')
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: code
+      - name: Checkout pages
+        uses: actions/checkout@v2
+        with:
+          ref: refs/heads/gh-pages
+          path: pages
+          fetch-depth: 0
```

Comment:
why do you need to fetch the full history of the pages?

Reply:
I.... don't remember.